src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -147,6 +147,13 @@ def _add_input_args(parser):
         help="The seed used to generate random values.",
     )
 
+    parser.add_argument(
+        "--synthetic-extra-inputs",
```

Comment:
why is this only available for synthetic data?

Reply:
Ah, I thought that's the only case where LLMInputs were generated. Made to work for all prompt sources!